apps/mull-ui/src/app/pages/event-page/event-page.tsx

```diff
@@ -18,8 +22,33 @@ export const EventPage = ({
   onBackButtonClick,
   onButtonClick,
   eventImageURL,
+  isReview = false,
 }: EventPageProps) => {
-  return (
+  let { id }: any = useParams();
+  const eventId = parseInt(id);
+
+  const GET_SPECIFIC_EVENT = gql`
+    query findSpecificEvent($eventId: Int!) {
+      event(id: $eventId) {
+        id
+        title
+        description
+        startDate
+        endDate
+        restriction
+      }
+    }
+  `;
+  const { loading, error, data } = useQuery(GET_SPECIFIC_EVENT, {
+    variables: { eventId },
```

Comment:
If the event doesn't exist in the database, a guard should redirect the user to a 404 page or something. This could be done in a separate ticket, however, to keep this pr small.